instrumentation/opentelemetry-instrumentation-elasticsearch/src/opentelemetry/instrumentation/elasticsearch/__init__.py

```diff
@@ -245,9 +245,11 @@ def wrapper(wrapped, _, args, kwargs):
                 if method:
                     attributes["elasticsearch.method"] = method
                 if body:
-                    attributes[SpanAttributes.DB_STATEMENT] = sanitize_body(
-                        body
-                    )
+                    # Don't set db.statement for bulk requests, as it can be very large
+                    if not isinstance(body, list):
```

Comment:
I would suggest to only accept dicts instead:

```suggestion
                    # Don't set db.statement for bulk requests, as it can be very large
                    if isinstance(body, dict):
```

since `body` can also be a string or bytes.

Not sure about the test, sorry.